### DISCUSSION TITLE
Create Orchard Core APIs Documentation

### DISCUSSION THREAD

**Discussion by hishamco** [OrchardCMS/OrchardCore#11078] [Ideas]
It would be useful if we can add `api.orchardcore.net` something similar to .NET APIs docs

**Comment by Piedone** [comment]
This would need inline docs in the Orchard APIs which idea [didn't really progress](https://github.com/OrchardCMS/OrchardCore/issues/5095).

**Comment by hishamco** [comment]
We already did a progress in the past and most of the newly added APIs are documented. May be we need sometime to revise and add the missing docs
